pkg/proc/ppc64le_arch.go

```diff
@@ -0,0 +1,237 @@
+package proc
+
+import (
+	"encoding/binary"
+	"fmt"
+	"strings"
+
+	"github.com/go-delve/delve/pkg/dwarf/frame"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/dwarf/regnum"
+)
+
+// This is the unconditional trap, the same mnemonic that both clang and gcc use
+// It's documented in Section C.6 Trap Mnemonics in the Power ISA Book 3
+var ppc64leBreakInstruction = []byte{0x08, 0x00, 0xe0, 0x7f}
+
+func PPC64LEArch(goos string) *Arch {
+	return &Arch{
+		Name:                             "ppc64le",
+		ptrSize:                          8,
+		maxInstructionLength:             4,
+		breakpointInstruction:            ppc64leBreakInstruction,
+		breakInstrMovesPC:                false, // TODO(alexsaezm) Verify if the break instruction moves the PC
+		derefTLS:                         false, // TODO(alexsaezm) Verify how the TLS behaves. Chapter 3.7 of the ELF V2 ABI Specification
```

Comment:
TODO (this variable is more about what Go does than TLS as specified by the ELF ABI: do we need to the reference the address in the TLS pseudoregister to get to the g struct?)